modules/ROOT/pages/apply-configure-xml-threat-task.adoc

```diff
@@ -9,31 +9,34 @@ endif::[]
+Applications processing XML requests are susceptible to attacks characterized by unusual inflation of elements, attributes, and nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic swings in the size of the application data often signal a security problem. The XML threat protection policy helps protect your applications from such intrusions.
 
-In the event Mule Runtime fails to detect an attack, you need to monitor and design your services architecture with layers of protection in addition to these policies.
+If you find that attacks on your Anypoint Platform setup are difficult to detect, design your services architecture with layers of protection in addition to JSON Threat Protection.
 
-== Configuration
+== Configuring Policy Parameters
+
+When you apply the XML Threat Protection policy to your API from the UI, the following parameters are displayed:
 
 [width="100%", cols="2,6,1,1"]
 |===
 s|Field s|Description s|Default s|Required
-|Maximum Node Depth|Specifies the maximum node depth allowed in the XML ^|0 ^|false
-|Maximum Attribute Count Per Element|Specifies the maximum number of attributes allowed for any element. Note that attributes used for defining namespaces are not counted ^|0 ^|false
-|Maximum Child Count|Specifies a limit on the maximum number of children allowed for any element in the XML document ^|0 ^|false
-|Maximum Text Length|Specifies a limit on the maximum length, in characters, of any text nodes present in the XML document ^|0 ^|false
-|Maximum Attribute Length|Specifies a limit on the maximum length, in characters, of any attributes for any element in the XML document ^|0 ^|false
-|Maximum Comment Length|Specifies a limit on the maximum number of comment characters in the XML document ^|0 ^|false
+|Maximum Node Depth|Specifies the maximum node depth of an XML document ^|0 ^|false
+|Maximum Attribute Count Per Element|Specifies the maximum number of attributes in an element. Note that attributes used for defining namespaces are not counted ^|0 ^|false
+|Maximum Child Count|Specifies the maximum number of children of an element in the XML document ^|0 ^|false
+|Maximum Text Length|Specifies the maximum length (in characters) of text nodes in the XML document ^|0 ^|false
+|Maximum Attribute Length|Specifies the maximum length (in characters) of an attribute in the XML document ^|0 ^|false
+|Maximum Comment Length|Specifies the maximum number of comment characters in the XML document ^|0 ^|false
 |===
 
 NOTE: Configuring a value of -1 means unlimited.
 
 == Example
 
+The following screenshot shows an example of the parameters configured for the XML Threat Protection policy:
```

Comment:
Ah! The policy name is init. capped here, so likely should be above.

Reply:
Yes changed that! Done! Thanks!